.gitlab-ci.yml

```diff
@@ -0,0 +1,102 @@
+stages:
+  - pull-github
+  - build
+  - deploy
+
+variables:
+  GIT_STRATEGY: clone			#This seems to have no effect also set in webinterface
+  GIT_DEPTH: 0					#This seems to have no effect also set in webinterface
```

Comment:
I am not sure `0` here is a correct value. The smallest one is `1` AFAIK. Can you confirm?
Perhaps this is why it does not work.

Reply:
0 means max depth we need that for git describe